js/package.json

```diff
@@ -2,73 +2,103 @@
   "name": "nglview-js-widgets",
   "version": "2.7.7",
   "description": "nglview-js-widgets",
-  "author": "Hai Nguyen <hainm.comp@gmail.com>, Alexander Rose <alexander.rose@weirdbyte.de>",
-  "license": "MIT",
-  "main": "lib/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/arose/nglview.git"
-  },
+  "keywords": [
+    "ipython",
+    "ipywidgets",
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension",
+    "molecular graphics",
+    "molecular structure",
+    "science",
+    "widgets"
+  ],
+  "homepage": "https://github.com/nglviewer/nglview/js",
   "bugs": {
-    "url": "https://github.com/arose/nglview/issues"
+    "url": "https://github.com/nglviewer/nglview/issues"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Alexander Rose",
+    "email": "alexander.rose@weirdbyte.de"
   },
+  "contributors": [
+    {
+      "name": "Hay Nguyen",
+      "email": "hainm.comp@gmail.com"
+    }
+  ],
```

Comment:
Please keep the original author list.

Reply:
I got test failures with the original list. It seems the `author` field only accepts one dictionary (https://nodejs.dev/learn/the-package-json-guide#author), while `contributors` accepts a list (https://nodejs.dev/learn/the-package-json-guide#contributors).

Reply:
What do you mean about 'test failures'?

Reply:
I added another GitHub workflow to test the plugin: https://github.com/nglviewer/nglview/pull/955/files#diff-d8e67b5b14ec13e17bc4b4a547de73afdb8ff0cb29acca3a3f23018dfd529694 and the eslint check step would fail with the previous format of `package.json`.

Reply:
It fails: https://github.com/nglviewer/nglview/runs/2229113065 because the `setup.py` relies on `package.json` defining one author only :/ Not sure what the best workaround is here.

Reply:
> the eslint check step would fail with the previous format of package.json.

How's about
- Keep the original format
- Ignore package.json in `.eslintignore`

Reply:
```
-    "name": "Alexander Rose",
-    "email": "alexander.rose@weirdbyte.de"
+    "name": "Alexander Rose, Hai Nguyen",
+    "email": "alexander.rose@weirdbyte.de, hainm.comp@gmail.com"
```

Reply:
It worked without touching `.eslintignore`. I might have been wrong as to why I had to make that change in the first place...

Reply:
By the way, I've removed the `AUTHORS` file, since it now seems unnecessary.